css/styles.css

```diff
@@ -0,0 +1,59 @@
+.row {
+  display: flex;
+}
+
+img {
+  width: 100%;
+}
+
+header {
+  padding-top: 0.1px;
+  padding-bottom: 0.1px;
+  background-color: #4c2676;
+}
+
+.header {
+  color: white;
+}
+
+.full-column {
+  flex-basis: 100%;
+}
+
+@media screen and (width > 567px) {
+  .column-half {
+    flex-basis: 50%;
+    margin-bottom: 15px;
+  }
+}
```

Comment:
The `row`, `column-full`, and `column-half` should already be in your `layout.css` file and you don't need to make them.

Also your `utility` classes should only do one job, so you should separate your `margin-bottom: 15px` and maybe make a different class for it in your `layout.css`. Below would be an example of making the utility class for margin bottom.

```
.margin-bot-15 {
  margin-bottom: 15px;
}
```